diff -urN ntop.org/plugins/pep/pep.c ntop/plugins/pep/pep.c
--- ntop.org/plugins/pep/pep.c	Mon Mar 26 19:30:37 2001
+++ ntop/plugins/pep/pep.c	Mon Mar 26 20:03:37 2001
@@ -52,7 +52,7 @@
 /*
  * a private instance of Perl
  */
-static PerlInterpreter * pep_perl = NULL;
+static PerlInterpreter * my_perl = NULL;
 
 
 /*
@@ -63,9 +63,9 @@
   /*
    * free the previously allocated perl instance
    */
-  perl_destruct (pep_perl);
-  perl_free (pep_perl);
-  pep_perl = NULL;
+  perl_destruct (my_perl);
+  perl_free (my_perl);
+  my_perl = NULL;
 
   traceEvent (TRACE_INFO, "\n");
   traceEvent (TRACE_INFO, "Thanks for having used PeP....\n");
@@ -176,7 +176,7 @@
   /*
    * compile the file
    */
-  perl_parse (pep_perl, NULL, pep_argc, pep_argv, NULL);
+  perl_parse (my_perl, NULL, pep_argc, pep_argv, NULL);
 
   /*
    * create a new variable, the %traffic hash
@@ -381,7 +381,7 @@
   /*
    * and then execute the script
    */
-  perl_run (pep_perl);
+  perl_run (my_perl);
 
    if (traffic)
     hv_undef (traffic);
@@ -410,8 +410,8 @@
   /*
    * initialization: allocate a private instance of the Perl Interpreter
    */
-  pep_perl = perl_alloc ();
-  perl_construct (pep_perl);
+  my_perl = perl_alloc ();
+  perl_construct (my_perl);
 
   return (plugininfo);
 }
