diff -Nur ntop-current.orig/gdchart0.94c/Makefile.in ntop-current/gdchart0.94c/Makefile.in
--- ntop-current.orig/gdchart0.94c/Makefile.in	Thu Oct 10 14:45:07 2002
+++ ntop-current/gdchart0.94c/Makefile.in	Thu Oct 10 14:46:46 2002
@@ -15,13 +15,13 @@
 	$(CC) -g -c price_conv.c
 
 gdc.o: gdc.c gdc.h
-	$(CC) -I$(GD_INCL) -I$(GDC_INCL) -g -c gdc.c
+	$(CC) -I$(GD_INCL) -I$(GDC_INCL) $(CFLAGS) -c gdc.c
 
 gdc_pie.o: $(GDC_INCL)/gdc.h $(GDC_INCL)/gdcpie.h gdc_pie.c
-	$(CC) -I$(GD_INCL) -I$(GDC_INCL) -g -c gdc_pie.c
+	$(CC) -I$(GD_INCL) -I$(GDC_INCL) $(CFLAGS) -c gdc_pie.c
 
 gdchart.o: $(GDC_INCL)/gdc.h $(GDC_INCL)/gdchart.h gdchart.c
-	$(CC) -I$(GD_INCL) -I$(GDC_INCL) -g -c gdchart.c
+	$(CC) -I$(GD_INCL) -I$(GDC_INCL) $(CFLAGS) -c gdchart.c
 
 $(GD_LIB)/libgd.a:
 	cd $(GD_LIB) ; $(MAKE) -f Makefile libgd.a
diff -Nur ntop-current.orig/ntop/Makefile.am ntop-current/ntop/Makefile.am
--- ntop-current.orig/ntop/Makefile.am	Thu Aug  8 17:19:16 2002
+++ ntop-current/ntop/Makefile.am	Thu Oct 10 15:04:14 2002
@@ -55,12 +55,6 @@
 
 SUBDIRS = . @PLUGINS@ @INTOP@
 
-DIST_COMMON = AUTHORS CONTENTS COPYING ChangeLog \
-              MANIFESTO NEWS PORTING  \
-              SUPPORT_NTOP.txt THANKS \
-              ntop.8 ntop.txt ntop.html \
-              ntop-cert.pem
-
 NTOPHTML = html html/*.js html/*.html  html/*.gif html/*.jpg html/*.ico html/*.png html/*.css html/statsicons \
            html/statsicons/flags html/statsicons/os \
            html/statsicons/flags/*.gif html/statsicons/os/*.gif
@@ -71,7 +65,9 @@
              configure configure.in \
              install-sh libtool ltconfig ltmain.sh \
              missing mkinstalldirs ntop-config.in \
-             vt.sed
+             vt.sed \
+	     CONTENTS MANIFESTO PORTING SUPPORT_NTOP.txt \
+	     ntop.txt ntop.html ntop-cert.pem
 
 WEBFILES       = `cat www/FILES`
 PLUGINSFILES   = `cat plugins/FILES`
diff -Nur ntop-current.orig/ntop/configure.in ntop-current/ntop/configure.in
--- ntop-current.orig/ntop/configure.in	Wed Oct  2 00:05:06 2002
+++ ntop-current/ntop/configure.in	Thu Oct 10 15:03:06 2002
@@ -538,6 +538,10 @@
   if test -f "/usr/include/ncurses.h"; then
     echo "checking for ncurses.h... yes"
     AC_DEFINE(HAVE_NCURSES_H)
+  else if test -f "/usr/include/ncurses/ncurses.h"; then
+    echo "checking for ncurses/ncurses.h... yes"
+    AC_DEFINE(HAVE_NCURSES_H)
+    INCS="${INCS} -I/usr/include/ncurses"
   else
     if test -f "/usr/include/curses.h"; then
       echo "checking for curses.h... yes"
@@ -547,6 +550,7 @@
      ac_disable_intop=yes
     fi
   fi
+  fi
 fi
 
 
@@ -876,7 +876,7 @@
 dnl>
 if test ".${PCAP_ROOT}" != .; then
   if test -d ${PCAP_ROOT} &&
-     test -r ${PCAP_ROOT}/libpcap.a &&
+     test -r ${PCAP_ROOT}/libpcap.a -o -r ${PCAP_ROOT}/libpcap.so &&
      test -r ${PCAP_ROOT}/pcap.h; then
        PCAP_ROOT=`cd ${PCAP_ROOT} && pwd`
        CORELIBS="${CORELIBS} -L${PCAP_ROOT} -lpcap"
@@ -885,7 +885,7 @@
   elif test -d ${PCAP_ROOT} &&
        test -d ${PCAP_ROOT}/lib &&
        test -d ${PCAP_ROOT}/include &&
-       test -r ${PCAP_ROOT}/lib/libpcap.a &&
+       test -r ${PCAP_ROOT}/lib/libpcap.a -o -r ${PCAP_ROOT}/lib/libpcap.so &&
        test -r ${PCAP_ROOT}/include/pcap.h; then
          PCAP_ROOT=`cd ${PCAP_ROOT} && pwd`
          CORELIBS="${CORELIBS} -L${PCAP_ROOT}/lib -lpcap"
@@ -895,7 +895,7 @@
        test -d ${PCAP_ROOT}/lib &&
        test -d ${PCAP_ROOT}/include &&
        test -d ${PCAP_ROOT}/include/pcap &&
-       test -r ${PCAP_ROOT}/lib/libpcap.a &&
+       test -r ${PCAP_ROOT}/lib/libpcap.a -o -r ${PCAP_ROOT}/lib/libpcap.so &&
        test -r ${PCAP_ROOT}/include/pcap/pcap.h; then
          PCAP_ROOT=`cd ${PCAP_ROOT} && pwd`
          CORELIBS="${CORELIBS} -L${PCAP_ROOT}/lib -lpcap"
@@ -914,7 +914,7 @@
 dnl> Older default installation 
 dnl>
 elif test -d /usr/local &&
-     test -r /usr/local/lib/libpcap.a &&
+     test -r /usr/local/lib/libpcap.a -o -r /usr/local/lib/libpcap.so &&
      test -r /usr/local/include/pcap.h; then
        PCAP_ROOT="/usr/local"
        CORELIBS="${CORELIBS} -L$PCAP_ROOT/lib -lpcap"
@@ -923,7 +923,7 @@
 dnl>
 dnl> installation used by most packagers
 dnl>
-elif test -r /usr/lib/libpcap.a &&
+elif test -r /usr/lib/libpcap.a -o -r /usr/lib/libpcap.so &&
      test -r /usr/include/pcap.h; then
        PCAP_ROOT="standard installation (/usr/lib and /usr/include)"
        CORELIBS="${CORELIBS} -lpcap"
@@ -932,7 +932,7 @@
 dnl> unusual and more complex installation used by some packagers
 dnl> (e.g. Red Hat Linux)
 dnl>
-elif test -r /usr/lib/libpcap.a &&
+elif test -r /usr/lib/libpcap.a -o -r /usr/lib/libpcap.so &&
      test -r /usr/include/pcap/pcap.h; then
        PCAP_ROOT="semi-standard installation (/usr/lib and /usr/include/pcap)"
        CORELIBS="${CORELIBS} -lpcap"
@@ -1327,8 +1331,8 @@
         AC_MSG_CHECKING([for libxxx.a in $1 and ssl.h in $2])
     fi
     if test -d $1 &&
-       test -r $1/libssl.a &&
-       test -r $1/libcrypto.a &&
+       test -r $1/libssl.a -o $1/libssl.so &&
+       test -r $1/libcrypto.a -o $1/libcrypto.so &&
        test -d $2 &&
        test -r $2/ssl.h; then
          if test ".$ac_enable_showopenssltests" = ".yes"; then
diff -Nur ntop-current.orig/ntop/intop/Makefile.am ntop-current/ntop/intop/Makefile.am
--- ntop-current.orig/ntop/intop/Makefile.am	Wed Jan  2 09:17:15 2002
+++ ntop-current/ntop/intop/Makefile.am	Thu Oct 10 15:04:32 2002
@@ -24,8 +24,6 @@
 # Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 #
 
-DIST_COMMON = Makefile.am Makefile.in intop.1
-
 DISTCLEANFILES = logger.db ntop.db ntop_pw.db intops
 CLEANFILES     = $(DISTCLEANFILES)
 
diff -Nur ntop-current.orig/ntop/plugins/Makefile.am ntop-current/ntop/plugins/Makefile.am
--- ntop-current.orig/ntop/plugins/Makefile.am	Tue Sep 10 16:13:19 2002
+++ ntop-current/ntop/plugins/Makefile.am	Thu Oct 10 15:04:41 2002
@@ -26,7 +26,6 @@
 
 SUBDIRS = . #pep
 
-DIST_COMMON = Makefile.am Makefile.in
 CLEANFILES  =
 EXTRA_DIST  =
 
@@ -93,7 +92,7 @@
 .libs/libicmpPlugin.so@SO_VERSION_PATCH@:
 	$(CC) -bundle -flat_namespace -undefined suppress -o .libs/libicmpPlugin.so@SO_VERSION_PATCH@ icmpPlugin.o
 
-icmpPlugin.so: .libs/libicmpPlugin.so@SO_VERSION_PATCH@
+icmpPlugin.so$(EXEEXT): .libs/libicmpPlugin.so@SO_VERSION_PATCH@
 	@ln -s .libs/libicmpPlugin.so icmpPlugin.so
 
 #.libs/librrdPlugin.so@SO_VERSION_PATCH@:
@@ -105,31 +104,31 @@
 .libs/libpdaPlugin.so@SO_VERSION_PATCH@:
 	$(CC) -bundle -flat_namespace -undefined suppress -o .libs/libpdaPlugin.so@SO_VERSION_PATCH@ pdaPlugin.o
 
-pdaPlugin.so: .libs/libpdaPlugin.so@SO_VERSION_PATCH@
+pdaPlugin.so$(EXEEXT): .libs/libpdaPlugin.so@SO_VERSION_PATCH@
 	@ln -s .libs/libpdaPlugin.so pdaPlugin.so
 
 .libs/libsflowPlugin.so@SO_VERSION_PATCH@:
 	$(CC) -bundle -flat_namespace -undefined suppress -o .libs/libsflowPlugin.so@SO_VERSION_PATCH@ sflowPlugin.o
 
-sflowPlugin.so: .libs/libsflowPlugin.so@SO_VERSION_PATCH@
+sflowPlugin.so$(EXEEXT): .libs/libsflowPlugin.so@SO_VERSION_PATCH@
 	@ln -s .libs/libsflowPlugin.so sflowPlugin.so
 
 .libs/liblastSeenPlugin.so@SO_VERSION_PATCH@:
 	$(CC) -bundle -flat_namespace -undefined suppress -o .libs/liblastSeenPlugin.so@SO_VERSION_PATCH@ lastSeenPlugin.o
 
-lastSeenPlugin.so: .libs/liblastSeenPlugin.so@SO_VERSION_PATCH@
+lastSeenPlugin.so$(EXEEXT): .libs/liblastSeenPlugin.so@SO_VERSION_PATCH@
 	@ln -s .libs/liblastSeenPlugin.so lastSeenPlugin.so
 
 .libs/libnfsPlugin.so@SO_VERSION_PATCH@:
 	$(CC) -bundle -flat_namespace -undefined suppress -o .libs/libnfsPlugin.so@SO_VERSION_PATCH@ nfsPlugin.o
 
-netflowPlugin.so: .libs/libnetflowPlugin.so@SO_VERSION_PATCH@
+netflowPlugin.so$(EXEEXT): .libs/libnetflowPlugin.so@SO_VERSION_PATCH@
 	@ln -s .libs/libnetflowPlugin.so netflowPlugin.so
 
 .libs/libnetflowPlugin.so@SO_VERSION_PATCH@:
 	$(CC) -bundle -flat_namespace -undefined suppress -o .libs/libnetflowPlugin.so@SO_VERSION_PATCH@ netflowPlugin.o
 
-nfsPlugin.so: .libs/libnfsPlugin.so@SO_VERSION_PATCH@
+nfsPlugin.so$(EXEEXT): .libs/libnfsPlugin.so@SO_VERSION_PATCH@
 	@ln -s .libs/libnfsPlugin.so nfsPlugin.so
 
 #.libs/librmonPlugin.so@SO_VERSION_PATCH@:
diff -Nur ntop-current.orig/ntop/plugins/pep/Makefile.am ntop-current/ntop/plugins/pep/Makefile.am
--- ntop-current.orig/ntop/plugins/pep/Makefile.am	Wed Jan  2 09:17:17 2002
+++ ntop-current/ntop/plugins/pep/Makefile.am	Thu Oct 10 14:57:58 2002
@@ -53,7 +53,7 @@
 #
 LIBS = # ${PEPLIBS}
 
-pep.so: pep.c
+pep.so$(EXEEXT): pep.c
 #	${CC} -shared ${libpep_la_OBJECTS} ${PEPLIBS} -Wl,-soname -Wl,libpep.so.0 -o pep.so
 	@${CC} -shared ${libpep_la_OBJECTS} ${PEPLIBS} -o pep.so
-	(cd .. && ln -fs pep/pep.so .)
\ No newline at end of file
+	(cd .. && ln -fs pep/pep.so .)
