--- ntop-2.2.95/plugins/Makefile.am.orig	Tue Jul 29 22:28:46 2003
+++ ntop-2.2.95/plugins/Makefile.am	Fri Dec 12 11:31:00 2003
@@ -59,7 +59,7 @@
 
 librrdPlugin_la_SOURCES = rrdPlugin.c
 librrdPlugin_la_LDFLAGS = -shared -release @PACKAGE_VERSION@ @DYN_FLAGS@
-librrdPlugin_la_LIBADD = -L../myrrd -lmyrrd
+librrdPlugin_la_LIBADD = -lrrd
 
 libpdaPlugin_la_SOURCES = pdaPlugin.c
 libpdaPlugin_la_LDFLAGS = -shared -release @PACKAGE_VERSION@ @DYN_FLAGS@
--- ntop-2.2.95/plugins/rrdPlugin.c.orig	Tue Sep 23 17:03:12 2003
+++ ntop-2.2.95/plugins/rrdPlugin.c	Fri Dec 12 11:32:12 2003
@@ -21,6 +21,7 @@
 /* This plugin works only with threads */
 
 /* #define RRD_DEBUG  8 */
+#define HAVE_RRD_ABERRANT_BEHAVIOR
 
 /*
 
@@ -493,7 +494,7 @@
 
     fillupArgv(argc, sizeof(argv)/sizeof(char*), argv);
     rrd_clear_error();
-    rc = rrd_graph(argc, argv, &calcpr, &x, &y);
+    rc = rrd_graph(argc, argv, &calcpr, &x, &y, NULL);
 
     calfree();
 
--- ntop-2.2.95/Makefile.am.orig	Fri Sep 12 00:54:51 2003
+++ ntop-2.2.95/Makefile.am	Fri Dec 12 11:27:03 2003
@@ -59,7 +59,7 @@
                        stamp-h.in
 
 
-SUBDIRS = myrrd . @PLUGINS@
+SUBDIRS = . @PLUGINS@
 
 ETTER_PASSIVE_DOWNLOAD = etter.passive.os.fp
 ETTER_PASSIVE = $(ETTER_PASSIVE_DOWNLOAD).gz
@@ -155,7 +155,7 @@
                      traffic.c    vendor.c      version.c      \
                      ntop_darwin.c
 
-libntop_la_DEPENDENCIES = myrrd/libmyrrd.a config.h
+libntop_la_DEPENDENCIES = config.h
 libntop_la_LIBADD       = 
 libntop_la_LDFLAGS      = $(AM_LDFLAGS) -release $(VERSION) -export-dynamic @DYN_FLAGS@
 
@@ -166,15 +166,12 @@
                            webInterface.c  xmldump.c
 
 
-libntopreport_la_DEPENDENCIES = myrrd/libmyrrd.a config.h
+libntopreport_la_DEPENDENCIES = config.h
 libntopreport_la_LIBADD       = 
 libntopreport_la_LDFLAGS      = $(AM_LDFLAGS) -release $(VERSION) -export-dynamic @DYN_FLAGS@
 
 man_MANS = ntop.8
 
-myrrd/libmyrrd.a:
-	cd myrrd; make
-
 #
 # Special stuff for xml auto-generation of code from the "doc":
 #
--- ntop-2.2.95/configure.in.orig	Tue Sep 23 17:35:05 2003
+++ ntop-2.2.95/configure.in	Fri Dec 12 11:29:11 2003
@@ -128,7 +128,7 @@
 AC_PREFIX_DEFAULT(${NTOPDIR:-/usr/local})
 
 # Set constants and defaults
-MYRRD=`pwd`/myrrd
+
 SO_VERSION_PATCH=
 
 AS=
@@ -2868,9 +2868,6 @@
 AC_DEFINE_UNQUOTED(CFG_DBFILE_DIR,"$CFG_DBFILE_DIR", [Database file directory])
 echo "    Database files are in $CFG_DBFILE_DIR"
 
-dnl> Append RRD finally
-NTOP_APPENDS([${MYRRD}], [${MYRRD}], [myrrd])
-
 echo
 echo "-------------------------------------------------------------------"
 echo
@@ -2898,7 +2895,6 @@
 AC_SUBST(SO_VERSION_PATCH)
 
 AC_CONFIG_FILES([Makefile])
-AC_CONFIG_FILES([myrrd/Makefile])
 
 if test ".${ac_disable_plugins}" != ".yes"; then
     AC_CONFIG_FILES([plugins/Makefile])
