--- ntop/plugins/Makefile.am	2004-11-11 10:12:52.000000000 +0000
+++ ntop.new/plugins/Makefile.am	2005-01-01 21:09:57.376911224 +0000
@@ -42,16 +42,6 @@
 #
 # The meat for ntop
 #
-noinst_PROGRAMS = \
-                  icmpPlugin.so \
-                  lastSeenPlugin.so \
-                  netflowPlugin.so \
-                  pdaPlugin.so \
-                  rrdPlugin.so \
-                  snmpPlugin.so \
-                  sflowPlugin.so \
-                  xmldumpPlugin.so
-
 lib_LTLIBRARIES = \
                   libicmpPlugin.la \
                   liblastSeenPlugin.la \
