--- ntop-2.2.95/configure.in.orig	Fri Dec 12 12:07:15 2003
+++ ntop-2.2.95/configure.in	Fri Dec 12 12:11:00 2003
@@ -1289,28 +1289,6 @@
 CPPFLAGS="${CPPFLAGS} -I/usr/local/include"
 LDFLAGS="${LDFLAGS} -L/usr/local/lib"
 
-
-
-echo
-echo "Test the compiler and setup CFLAGS..."
-echo
-case "$CFLAGS" in
-    *-g* )
-        ;;
-    * )
-        CFLAGS="$CFLAGS -g"
-        AC_MSG_CHECKING([if compiler accepts -g (debugging information)])
-        oCFLAGS=$CFLAGS
-        CFLAGS="-g $CFLAGS"
-        AC_TRY_COMPILE( ,
-                       return 0 ,
-                       AC_MSG_RESULT(yes),
-                       CFLAGS="${oCFLAGS}"
-                       AC_MSG_RESULT(no)
-                      )
-        ;;
-esac
-
 AC_MSG_CHECKING([if compiler accepts -W (warning flags)])
 oCFLAGS=$CFLAGS
 CFLAGS="$CFLAGS -Wshadow -Wpointer-arith -Wmissing-prototypes -Wmissing-declarations -Wnested-externs"
