--- Makefile.am.orig	Thu Apr  4 12:46:12 2002
+++ Makefile.am	Thu Apr  4 13:24:41 2002
@@ -57,31 +57,31 @@
 #		  librmonPlugin.la libwapPlugin.la
 
 libicmpPlugin_la_SOURCES = icmpPlugin.c
-libicmpPlugin_la_LDFLAGS = -shared -version-info @NTOP_VERSION_INFO@ @DYN_FLAGS@
+libicmpPlugin_la_LDFLAGS = -shared
 
 #librrdPlugin_la_SOURCES = rrdPlugin.c
-#librrdPlugin_la_LDFLAGS = -shared -version-info @NTOP_VERSION_INFO@ @DYN_FLAGS@
+#librrdPlugin_la_LDFLAGS = -shared
 
 libpdaPlugin_la_SOURCES = pdaPlugin.c
-libpdaPlugin_la_LDFLAGS = -shared -version-info @NTOP_VERSION_INFO@ @DYN_FLAGS@
+libpdaPlugin_la_LDFLAGS = -shared
 
 libsflowPlugin_la_SOURCES = sflowPlugin.c
-libsflowPlugin_la_LDFLAGS = -shared -version-info @NTOP_VERSION_INFO@ @DYN_FLAGS@
+libsflowPlugin_la_LDFLAGS = -shared
 
 liblastSeenPlugin_la_SOURCES = lastSeenPlugin.c
-liblastSeenPlugin_la_LDFLAGS = -shared -version-info @NTOP_VERSION_INFO@ @DYN_FLAGS@
+liblastSeenPlugin_la_LDFLAGS = -shared
 
 libnetflowPlugin_la_SOURCES = netflowPlugin.c
-libnetflowPlugin_la_LDFLAGS = -shared -version-info @NTOP_VERSION_INFO@ @DYN_FLAGS@
+libnetflowPlugin_la_LDFLAGS = -shared
 
 libnfsPlugin_la_SOURCES = nfsPlugin.c
-libnfsPlugin_la_LDFLAGS = -shared -version-info @NTOP_VERSION_INFO@ @DYN_FLAGS@
+libnfsPlugin_la_LDFLAGS = -shared
 
 #librmonPlugin_la_SOURCES = rmonPlugin.c rmon.h
-#librmonPlugin_la_LDFLAGS = @SNMPLIBS@ -shared -version-info @NTOP_VERSION_INFO@ @DYN_FLAGS@
+#librmonPlugin_la_LDFLAGS = @SNMPLIBS@ -shared
 
 #libwapPlugin_la_SOURCES = wapPlugin.c
-#libwapPlugin_la_LDFLAGS = -shared -version-info @NTOP_VERSION_INFO@ @DYN_FLAGS@
+#libwapPlugin_la_LDFLAGS = -shared
 
 
 #
@@ -90,58 +90,40 @@
 #
 
 
-.libs/libicmpPlugin.so@SO_VERSION_PATCH@:
-	cc -bundle -flat_namespace -undefined suppress -o .libs/libicmpPlugin.so@SO_VERSION_PATCH@ icmpPlugin.o
+.libs/libicmpPlugin.so:
+	cc -bundle -flat_namespace -undefined suppress -o .libs/libicmpPlugin.so icmpPlugin.o
 
-icmpPlugin.so: .libs/libicmpPlugin.so@SO_VERSION_PATCH@
-	@ln -s .libs/libicmpPlugin.so icmpPlugin.so
+#.libs/librrdPlugin.so:
+#	cc -bundle -flat_namespace -undefined suppress -o .libs/librrdPlugin.so rrdPlugin.o
 
-#.libs/librrdPlugin.so@SO_VERSION_PATCH@:
-#	cc -bundle -flat_namespace -undefined suppress -o .libs/librrdPlugin.so@SO_VERSION_PATCH@ rrdPlugin.o
-
-#rrdPlugin.so: .libs/librrdPlugin.so@SO_VERSION_PATCH@
+#rrdPlugin.so: .libs/librrdPlugin.so
 #	@ln -s .libs/librrdPlugin.so rrdPlugin.so
 
-.libs/libpdaPlugin.so@SO_VERSION_PATCH@:
-	cc -bundle -flat_namespace -undefined suppress -o .libs/libpdaPlugin.so@SO_VERSION_PATCH@ pdaPlugin.o
-
-pdaPlugin.so: .libs/libpdaPlugin.so@SO_VERSION_PATCH@
-	@ln -s .libs/libpdaPlugin.so pdaPlugin.so
-
-.libs/libsflowPlugin.so@SO_VERSION_PATCH@:
-	cc -bundle -flat_namespace -undefined suppress -o .libs/libsflowPlugin.so@SO_VERSION_PATCH@ sflowPlugin.o
-
-sflowPlugin.so: .libs/libsflowPlugin.so@SO_VERSION_PATCH@
-	@ln -s .libs/libsflowPlugin.so sflowPlugin.so
-
-.libs/liblastSeenPlugin.so@SO_VERSION_PATCH@:
-	cc -bundle -flat_namespace -undefined suppress -o .libs/liblastSeenPlugin.so@SO_VERSION_PATCH@ lastSeenPlugin.o
-
-lastSeenPlugin.so: .libs/liblastSeenPlugin.so@SO_VERSION_PATCH@
-	@ln -s .libs/liblastSeenPlugin.so lastSeenPlugin.so
+.libs/libpdaPlugin.so:
+	cc -bundle -flat_namespace -undefined suppress -o .libs/libpdaPlugin.so pdaPlugin.o
 
-.libs/libnfsPlugin.so@SO_VERSION_PATCH@:
-	cc -bundle -flat_namespace -undefined suppress -o .libs/libnfsPlugin.so@SO_VERSION_PATCH@ nfsPlugin.o
+.libs/libsflowPlugin.so:
+	cc -bundle -flat_namespace -undefined suppress -o .libs/libsflowPlugin.so sflowPlugin.o
 
-netflowPlugin.so: .libs/libnetflowPlugin.so@SO_VERSION_PATCH@
-	@ln -s .libs/libnetflowPlugin.so netflowPlugin.so
+.libs/liblastSeenPlugin.so:
+	cc -bundle -flat_namespace -undefined suppress -o .libs/liblastSeenPlugin.so lastSeenPlugin.o
 
-.libs/libnetflowPlugin.so@SO_VERSION_PATCH@:
-	cc -bundle -flat_namespace -undefined suppress -o .libs/libnetflowPlugin.so@SO_VERSION_PATCH@ netflowPlugin.o
+.libs/libnfsPlugin.so:
+	cc -bundle -flat_namespace -undefined suppress -o .libs/libnfsPlugin.so nfsPlugin.o
 
-nfsPlugin.so: .libs/libnfsPlugin.so@SO_VERSION_PATCH@
-	@ln -s .libs/libnfsPlugin.so nfsPlugin.so
+.libs/libnetflowPlugin.so:
+	cc -bundle -flat_namespace -undefined suppress -o .libs/libnetflowPlugin.so netflowPlugin.o
 
-#.libs/librmonPlugin.so@SO_VERSION_PATCH@:
-#	cc -bundle -flat_namespace -undefined suppress -o .libs/librmonPlugin.so@SO_VERSION_PATCH@ rmonPlugin.o
+#.libs/librmonPlugin.so:
+#	cc -bundle -flat_namespace -undefined suppress -o .libs/librmonPlugin.so rmonPlugin.o
 
-#rmonPlugin.so: .libs/librmonPlugin.so@SO_VERSION_PATCH@
+#rmonPlugin.so: .libs/librmonPlugin.so
 #	@ln -s .libs/librmonPlugin.so rmonPlugin.so
 
-#.libs/libwapPlugin.so@SO_VERSION_PATCH@:
-#	cc -bundle -flat_namespace -undefined suppress -o .libs/libwapPlugin.so@SO_VERSION_PATCH@ wapPlugin.o
+#.libs/libwapPlugin.so:
+#	cc -bundle -flat_namespace -undefined suppress -o .libs/libwapPlugin.so wapPlugin.o
 
-#wapPlugin.so: .libs/libwapPlugin.so@SO_VERSION_PATCH@
+#wapPlugin.so: .libs/libwapPlugin.so
 #	@ln -s .libs/libwapPlugin.so wapPlugin.so
 
 install-data-local:
