--- Makefile.am.orig	Sat Mar 30 00:05:08 2002
+++ Makefile.am	Thu Apr  4 12:44:48 2002
@@ -128,7 +128,6 @@
 
 libntop_la_DEPENDENCIES = config.h
 libntop_la_LIBADD       = $(CORELIBS)
-libntop_la_LDFLAGS      = -version-info @NTOP_VERSION_INFO@ -release @NTOP_RELEASE@ -export-dynamic @DYN_FLAGS@
 
 # Archive for http representation, or the 'viewer'
 libntopreport_la_SOURCES = admin.c emitter.c \
@@ -142,7 +141,6 @@
 
 libntopreport_la_DEPENDENCIES = config.h
 libntopreport_la_LIBADD       = $(MORELIBS)
-libntopreport_la_LDFLAGS      = -version-info @NTOP_VERSION_INFO@ -release @NTOP_RELEASE@ -export-dynamic @DYN_FLAGS@
 
 man_MANS = ntop.8
 
